topics/mongodb/index.md

```diff
@@ -5,9 +5,9 @@ display_name: MongoDB
 github_url: https://github.com/mongodb
 logo: mongodb.png
 released: February 11, 2009
-short_description: MongoDB is a NoSQL cross-platform database.
+short_description: MongoDB is an open source NoSQL document-oriented database management system.
```

Comment:
Thoughts on removing "management system" ? I think of MongoDB as a database and something like AWS Management Console a management system.

Reply:
Good point, though MySQL and Postgress  include the terminology, and they are not a management console either. The term is commonly used in "traditional" database systems, giving birth to the acronym RDBMS.

Reply:
Thanks for the explanation, TIL! 

With that, this LGTM.